app/views/results/common/_flexible_marks_summary_table.html.erb

```diff
@@ -26,7 +26,7 @@
     <tr class="<%= cycle('mark_summary_holderA', 'mark_summary_holderB') %>" id="<%= "test_script_summary" %>">
         <td><%=t("summary_table.test_script_title")%></td>
         <td><%= h(@assignment.total_test_script_marks * 1.0) %></td>
-        <td><span id="test_script_summary_mark"><%= h(@submission.result.get_total_test_script_marks * 1.0) %></span> / <%=  h(@assignment.total_test_script_marks * 1.0) %></td>
+        <td><span id="test_script_summary_mark"><%= h(@result.get_total_test_script_marks * 1.0) %></span> / <%=  h(@assignment.total_test_script_marks * 1.0) %></td>
```

Comment:
@alexvail the issue was not really what you said, but the fact that the submission model doesn't have a "result" method/attribute - see the `models/submission.rb` file. But anyway, the change is good.